components/BLE/COMPONENT_BlueNRG_2/BlueNrg2HCIDriver.cpp

```diff
@@ -0,0 +1,532 @@
+/*
+ * Copyright (c) 2017-2020 ARM Limited
+ * Copyright (c) 2017-2020 STMicroelectronics
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+
+// drivers
+#include "drivers/DigitalOut.h"
+#include "drivers/SPI.h"
+#include "drivers/InterruptIn.h"
+
+// platform
+#include "platform/mbed_wait_api.h"
+
+// FEATURE_BLE/targets/TARGET_CORDIO
+#include "CordioBLE.h"
+#include "CordioHCIDriver.h"
+#include "CordioHCITransportDriver.h"
+#include "hci_api.h"
+#include "hci_cmd.h"
+#include "hci_core.h"
+#include "dm_api.h"
+#include "bstream.h"
+#include "hci_mbed_os_adaptation.h"
+
+// rtos
+#include "Thread.h"
```

Comment:
lets use prefixes like on line 22, rtos/header.h

Reply:
The structure of the folder is the same as components/BLE/COMPONENT_BlueNRG_MS